src/daomop/vospace.py

```diff
@@ -33,13 +32,15 @@ def my_func(*args, **kwargs):
 
```

Comment:
The vospace module is deprecated.  usage of daomop.vospace may can  be replaced by direct access vos.Client() calls?